src/socket.io/categories.js

```diff
@@ -203,10 +203,10 @@ function ignoreOrWatch(fn, socket, cid, callback) {
 
 			async.each(cids, function (cid, next) {
 				fn(socket.uid, cid, next);
-			}, next);
+			}, next(null, cids));
```

Comment:
can you just make cids a local variable to the function and keep these next calls the way they are?